extensions/opentelemetry/opentelemetry-exporter-otlp/runtime/src/main/java/io/quarkus/opentelemetry/exporter/otlp/runtime/OtlpExporterConfig.java

```diff
@@ -48,5 +48,15 @@
         @ConfigItem(defaultValue = "10S")
         public Duration exportTimeout;
 
+        /**
+         * Compression method to be used by exporter to compress the payload.
+         * <p>
+         * See <a href=
+         * "https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/exporter.md#opentelemetry-protocol-exporter">
+         * Configuration Options</a> for the supported compression methods.
+         */
+        @ConfigItem()
+        public Optional<String> compression;
```

Comment:
Added compression as an optional string, and it points to the spec configuration options. The SDK already validates the compression option and it throws an exception telling which codec is supported.

Reply:
Fantastic!